kolibri/auth/test/test_permissions.py

```diff
@@ -94,7 +97,7 @@ class ClassroomPermissionsTestCase(TestCase):
 
     def setUp(self):
         self.data = create_dummy_facility_data()
-        self.member = self.data["learners_one_group"][0][0][0]
+        self.member = self.data["learners_one_group"][0][0]
```

Comment:
There's no reason to go back and change it now since it works just fine, but in the future I think I prefer non-DRY setups for test methods vs an opaque function to set it up test data. Turns out there are a handful of projects (like [factoryboy](https://factoryboy.readthedocs.org/en/latest/)) for generating test data as well, so we might look in to them (I haven't yet).